flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecDeduplicate.java

```diff
@@ -89,6 +91,17 @@
                                     + "but there will be additional overhead."
                                     + "Default is true.");
 
+    @Experimental
+    public static final ConfigOption<Boolean>
+            TABLE_EXEC_DEDUPLICATE_MINIBATCH_ALL_CHANGELOG_ENABLED =
+                    ConfigOptions.key("table.exec.deduplicate.mini-batch.all.change-log.enabled")
+                            .booleanType()
+                            .defaultValue(true)
+                            .withDescription(
+                                    "Set whether send all change log for row-time mini-batch."
+                                            + "If false, Flink will only send the latest change log to downstream like proc-time."
```

Comment:
If downstream is: https://ci.apache.org/projects/flink/flink-docs-master/docs/dev/table/concepts/versioned_tables/#versioned-table-views
This optimization can not be enabled.

We should add description here.